### PR TITLE
Update rpc_core.rb to add evasion modules statistics

### PR DESCRIPTION
**Pull request by EmilioPanti**
Simple fix to add statistics regarding evasion modules

### CODE REVIEW

**Review by gwillcox-r7 — APPROVED** [review]

File: lib/msf/core/rpc/v10/rpc_core.rb [hunk 1]
```diff
@@ -137,7 +137,8 @@ def rpc_module_stats
       'post'      => framework.stats.num_post,
       'encoders'  => framework.stats.num_encoders,
       'nops'      => framework.stats.num_nops,
-      'payloads'  => framework.stats.num_payloads
+      'payloads'  => framework.stats.num_payloads,
+      'evasions'  => framework.stats.num_evasion
```
Comment (gwillcox-r7) [inline lib/msf/core/rpc/v10/rpc_core.rb:141]: Actually one small thing, but we should update the RDoc here to mention that evasions can now be returned and explain this properly.

**Review by gwillcox-r7 — CHANGES_REQUESTED** [review]

File: lib/msf/core/rpc/v10/rpc_core.rb (hunk 1 above)
Comment (EmilioPanti) [reply lib/msf/core/rpc/v10/rpc_core.rb:144]: I have updated the api descriptions to mention that the statistics on the evasion modules are now being returned as well. 
I'm not sure you meant this, if not, could you please explain more specifically what I should update?

**Comment by jmartin-r7** [comment]
@msjenkins-r7 test this please.

**Review by gwillcox-r7 — APPROVED** [review]

**Comment by gwillcox-r7** [comment]
# Release Notes
` rpc_core.rb` has been updated so that it now reports the number of evasion modules within Metasploit. Previously this statistic wasn't being reported, whilst other statistics like number of exploit modules, auxiliary modules, and payloads were.

**Comment by gwillcox-r7** [comment]
Thanks for providing this update @EmilioPanti and congrats on your first PR to Metasploit!🥳

**Comment by EmilioPanti** [comment]
Thanks @gwillcox-r7 , I hope to contribute again in the future!

**Pull request merged**